lib/spec/runners/operations.ex

```diff
@@ -162,4 +162,16 @@ defmodule OperationsTestRunner do
         assert nil == post
     end
   end
+
+  defp handle_case("bls_to_execution_change", pre, operation, post, _case_dir) do
+    result = Operations.process_bls_to_execution_change(pre, operation)
+
+    case result do
+      {:ok, new_state} ->
+        assert new_state == post
```

Comment:
```suggestion
        assert {:ok, state} = result
        assert Diff.diff(state, post) == :unchanged
```

Reply:
This shouldn't be needed once it's synced with main, since #403 was merged